### DISCUSSION TITLE
Standalone Component support

### DISCUSSION THREAD

**Discussion by Elias-Graf** [ghiscoding/Angular-Slickgrid#1184] [General]
(When) will this library be migrated to standalone components?

**Comment by ghiscoding** [comment]
I don't intend to make it more standalone than it is now with what I've done with Slickgrid-Universal by making many pieces optional (GraphQL, OData, Excel Export, ...).

**Reply by Elias-Graf** [reply]
Oh, okay. Just to be sure, we're both talking about angular standalone components (https://angular.io/guide/standalone-components), not more npm modules etc, right?

**Reply by ghiscoding** [reply]
ah yeah it's not the same subject then, the answer would be that I have no time to look into this and I have not and probably won't use this new Angular approach for a long while. So it would probably only happen if someone like you would contribute a Pull Request.
